tests/integration/test_logcollector/test_configuration/test_basic_configuration_age.py

```diff
@@ -16,7 +16,7 @@
 from wazuh_testing.tools.services import control_service
 import subprocess as sb
 
-LOGCOLLECTOR_DAEMON = "wazuh-logcollector"
+LOGCOLLECTOR_DAEMON = "wazuh-modulesd"
```

Comment:
This variable is declared in several tests. I think we can move it to `logcollector.py` and import this variable in the tests.